app/views/cases/attachments/new.html.slim

```diff
@@ -0,0 +1,37 @@
+- content_for :page_title do
+  = t('page_title.add_request_attachment_page', case_number: @case.number)
+
+= GovukElementsErrorsHelper.error_summary @case,
+  "#{pluralize(@case.errors.count, t('common.error'))} #{ t('common.summary_error')}", ""
+
+- content_for :heading
+  = t('cases.attachments.upload_request_files_heading')
+
+- content_for :sub_heading
+
+= link_to("Back", case_path(@case), class: 'govuk-back-link')
+
+span.visually-hidden
+  = t('common.case.header_case_number')
+  = @case.number
+
+= render partial: 'layouts/header'
+
+
+= form_for @case, as: :case_attachment, url: case_attachments_path(@case), method: :post do |f|
+  .form-group
+    .grid-row
+      .column-two-thirds
+        p.form-hint
+          = t('cases.attachments.upload_hint_copy')
+    .grid-row
+      section.case-main
+        = render partial: 'shared/dropzone_form',
```

Comment:
Is this the same upload form used everywhere or are there others, just asking as we have some keyboard accessibility issues. If it's the same we can fix that later as there is a ticket for it.

Reply:
@digitalali-moj  yes, it is the same upload form used everywhere